### PR TITLE
feat: add generic monitoring (MN-299)

### PR DESCRIPTION
**Pull request by deepakglobant**

### CODE REVIEW

File: app.config.js [hunk 1]
```diff
@@ -280,6 +280,6 @@ export default {
       palette,
     },
     ...app,
-    ...eas.build.preview.env,
+    ...eas.build.development.env,
```
Comment (rohitsharma120582) [inline app.config.js:283]: Point to `preview` env as this build will be deployed on Preview environment

File: src/main.js [hunk 2]
```diff
@@ -1,32 +1,30 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```
Comment (rohitsharma120582) [inline src/main.js:1]: `useEffect` is not being used at the component level. Plz remove it.

**Review by rohitsharma120582 — CHANGES_REQUESTED** [review]

File: src/main.js (hunk 2 above)
Comment (deepakglobant) [reply src/main.js]: Resolved.

File: app.config.js (hunk 1 above)
Comment (deepakglobant) [reply app.config.js]: Resolved.

**Review by rohitsharma120582 — APPROVED** [review]

**Review by souyahia-monk — APPROVED** [review]

**Pull request merged**